dist/api_data.py

```diff
@@ -1094,6 +1094,11 @@ def __ge__(self, other):
         closed. This value is inherited from ::wiredtiger_open
         \c cache_cursors''',
         type='boolean'),
+    Config('cache_max_wait_ms', '0', r'''
```

Comment:
Will the return error code indicate whether the operation is successful or not or whether cache was evicted or not?

Reply:
No, the user only gets a rollback error.

Reply:
I agree it a bit strange here. May be we should return rollback not cache full in this case.

Reply:
I think we should keep as it is and let mongodb handle this the same way as cache full case.

Reply:
My concern here is if the operation is successful, and cache is full, there is no way we could indicate to server that the operation is successful.